build-system/tasks/check-links.js

```diff
@@ -62,7 +62,8 @@ function runLinkChecker(markdownFile) {
     baseUrl : 'file://' + path.dirname(path.resolve((markdownFile)))
   };
 
-  markdownLinkCheck(filteredMarkdown, opts, function(error, results) {
+  return markdownLinkCheck(filteredMarkdown, opts)
+  .then(function(error, results) {
```

Comment:
Good catch. That was the part I missed when I switched to using bluebird promises. Fixed.